src/components/MultipleAnswer.tsx

```
@@ -0,0 +1,61 @@
+        <Checkbox
+          key={`check-${item}`}
+          value={item}
+          label={<Text typo="body1">{item}</Text>}
+          style={{ color: color.textBlack }}
+        />
+      ))}
+      <Checkbox
+        key={`check-기타`}
+        value="기타"
+        onClick={() => {
+          textInputRef.current && textInputRef.current.focus();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)